src/docs/GetStarted/Developers/Installation/index.mdx

```diff
@@ -7,8 +7,8 @@ order: 1
 
 Installing Mozaic require `node v10 +`
 
-As for now the `@mozaic-ds/**` repos are public registries on NPM.
-You need to be granted and access to the repos.
+As for now, the `@mozaic-ds/**` repositories are public registries on NPM.
+You need to be granted and access to the repositories.
```

Comment:
Don't you want to say: 
You need to be granted to have access to the repositories.

❓